src/AeroSharp/AeroSharp.csproj

```diff
@@ -2,8 +2,29 @@
 
     <PropertyGroup>
         <TargetFramework>netstandard2.1</TargetFramework>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <Version>0.1.0</Version>
+        <Authors>Wayfair</Authors>
+        <Description>Wrapper around the .NET Aerospike client that provides a variety of methods for storing and retrieving data on Aerospike</Description>
+        <Copyright>Wayfair ©2021</Copyright>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <PackageProjectUrl>https://github.com/wayfair-incubator/AeroSharp</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/wayfair-incubator/AeroSharp</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>aerospike, aerosharp, serialization, policy validation</PackageTags>
     </PropertyGroup>
 
+    <PropertyGroup>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    </PropertyGroup>
```

Comment:
Nit: this is unrelated to NuGet packaging and shouldn't be introduced in this PR. Treating build warnings as build errors is something we may want to introduce later, but lets leave it out for now.